src/main/java/org/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -386,7 +390,23 @@ private void parseJabRefComment(Map<String, String> meta) {
             } catch (ParseException ex) {
                 parserResult.addException(ex);
             }
+        } else if (comment.substring(0, Math.min(comment.length(), MetaData.META_FLAG_VERSION_010.length())).equals(MetaData.META_FLAG_VERSION_010)) {
+            parseCommentToJson(comment, meta);
+        }
+    }
+
+    private JsonObject parseCommentToJson(String comment, Map<String, String> meta) {
+        Pattern pattern = Pattern.compile("\\{.*}", Pattern.DOTALL);
+        Matcher matcher = pattern.matcher(comment);
```

Comment:
I think, just doing `comments.substring(MetaData.META_FLAG_VERSION_010)` should to the trick. - No need for more checking here.